src/explore-education-statistics-common/src/modules/charts/components/ChartRenderer.tsx

```diff
@@ -63,14 +63,29 @@ function ChartRenderer({
     }
   }, [id, props]);
 
+  const { type } = props;
+  const { footnotes } = meta;
+  if (type === 'map') {
+    const boundaryFootnoteLabel = `The boundary data used in this map includes${meta.boundaryLevels
+      .map((value, index, array) => {
+        let separator = index === 0 ? ' ' : ', ';
+        if (array.length - 1 === index) {
+          separator = ' and ';
+        }
+        return `${separator}${value.label}`;
+      })
+      .join('')}`;
+    footnotes.push({ id: `map-footnote`, label: boundaryFootnoteLabel });
```

Comment:
I don't think this is the right place to be adding a default footnote. My inclination is that this should be added in the backend.

Probably requires a quick discussion to determine what the best place to do this is.